ethology/trackers/tracker.py

```diff
@@ -0,0 +1,98 @@
+"""Trackers module."""
+
+import os
+from typing import Literal
+
+import torch
+
+from .cotracker.cotracker.utils.visualizer import (
+    Visualizer,
+    read_video_from_path,
```

Comment:
Using `read_video_from_path` api from **cotracker** repository

I checked the source code, they use `imageio` with `numpy` to read the video file, which I think is similar to that in movement (which uses `sleap_io` that uses `imageio` as well), so maybe reducing a step

Reply:
sorry what do you mean with reducing a step?

Reply:
By reducing a step, I mean reducing an import step, directly using from cotracker to read the video, instead of using movement's way of reading, which also has the same, but a deeper hierarchy. Let me know which is the preferred way to read the video.